htdocs/societe/card.php

```diff
@@ -3224,7 +3224,7 @@ function GRVAT(a, u, p, myafm) {
 
   $.ajax({
 	type: "GET",
-	url: "/societe/checkvat/checkVatGr.php",
+	url: './checkvat/checkVatGr.php',
```

Comment:
To avoidb troubles with path, it is recommended to use absolute path so here
DOL_URL_ROOT.'/societe/...

Reply:
Done